src/agents/voice/models/openai_stt.py

```diff
@@ -65,7 +65,8 @@ async def _wait_for_event(
         if evt_type in expected_types:
             return evt
         elif evt_type == "error":
-            raise Exception(f"Error event: {evt.get('error')}")
+            # fixed: changed generic exception to specific sttwebsocketconnectionerror
+            raise STTWebsocketConnectionError(f"Error event: {evt.get('error')}")
```

Comment:
using generic Exception may not be good but using STTWebsocketConnectionError here is not relevant.